### DISCUSSION TITLE
https://blog.3place.cn/3place/sszs/SSZS008.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#109] [General]
# https://blog.3place.cn/3place/sszs/SSZS008.html

https://blog.3place.cn/3place/sszs/SSZS008.html

**Comment by picklecai** [comment]
有文无质，不是历来如此。而是每一种文学形式发展久了以后的表现。
骈文到唐朝，古文到明清。现在老师教小孩子所谓“答题技巧”，优秀作文模板，都是这类。

**Comment by picklecai** [comment]
怪不得历史书上说胡适、陈独秀开启白话运动、新文化运动。果然各有厉害之处。适之适之，独秀独秀！真了不起！
